conda-forge.yml

```diff
@@ -3,10 +3,6 @@ provider:
   linux_aarch64: azure
 build_platform:
   linux_aarch64: linux_64
-# NOTE: For binary redists os_version should match oldest supported glibc
-os_version:
-  linux_64: alma8
-  linux_aarch64: alma8
```

Comment:
Note that it's not necessary to set the os_version if we are checking the binaries for glibc symbols.

Reply:
Gonna revert this until we get approval from conda-forge/core at a future meeting. Or maybe just wait until that meeting before merging this PR.

Reply:
Discussing this tomorrow at meeting.